scala/lambdas/preingest-tdr-package-builder/src/main/scala/uk/gov/nationalarchives/preingesttdrpackagebuilder/Lambda.scala

```diff
@@ -114,7 +133,6 @@ class Lambda extends LambdaRunner[Input, Output, Config, Dependencies]:
             .toStreamBuffered[IO](bufferSize)
             .flatMap(bf => Stream.chunk(Chunk.byteBuffer(bf)))
             .through(byteStreamParser[IO])
-            .through(fs2Decoder[IO, TDRMetadata])
```

Comment:
It might be possible. I did try but I kept ending up with an empty Stream with the Publisher throwing exceptions so I gave up and did it like this :smile: